pkg/operators/kubemanager/kubemanager.go

```diff
@@ -475,7 +506,11 @@ func (m *KubeManagerInstance) PreStart(gadgetCtx operators.GadgetContext) error
 }
 
 func (m *KubeManagerInstance) Start(gadgetCtx operators.GadgetContext) error {
-	return nil
+	if m.containersPublisher == nil {
+		return nil
+	}
+
+	return m.containersPublisher.PublishContainers(true, false, nil, "")
```

Comment:
What does it mean to pass `""` on the container name? Is it not possible to get notifications for some specific containers? I think it should be generic. What about making PublishContainers receive `containercollection.ContainerSelector`?